release-notes/README.md

```diff
@@ -4,6 +4,26 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## August 2, 2022
+
+### Android Target 1.3.0
+
+Added support for Target tnt ID and session ID getter and setter APIs to enable cross-channel sessions.
```

Comment:
Former. There are getters and setters APIs for tnt ID - setTntId, getTntId and also for sessionId - getSessionId and setSessionId.

Reply:
Okay.

Let's try this instead then - "Added getter and setter APIs for Target tnt IDs and session IDs to enable cross-channel sessions."

Since I think that avoids any ambiguity in the statement.

Reply:
@calebk1 Let me know if we need to change the section here for the sake of clarity.

Reply:
Yeah, I would change it here (and in the other places where we have this sentence).

Reply:
@calebk1 I've pushed the changes as discussed. Please review and let me know if it seems fine.